.env

```diff
@@ -0,0 +1,10 @@
+MYSQLDB_USER=root
+MYSQLDB_ROOT_PASSWORD=12345678
+MYSQLDB_DATABASE=book_store
+
+MYSQL_LOCAL_PORT=3307
+MYSQL_DOCKER_PORT=3306
+
+SPRING_LOCAL_PORT=8088
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```

Comment:
Don’t push .env to GH. Remove this file from the remote repository. Better push .env.template with variables without values